bot/subscription/__main__.py

```diff
@@ -30,11 +56,16 @@ async def start(update: Update, context: ContextTypes.DEFAULT_TYPE) -> int:
         )
         return ConversationHandler.END
     user_id = update.message.chat.id
-    context.user_data["_id"] = user_id
-    current_user = db.subscribers.find_one({"_id": user_id})
+    context.user_data["id"] = user_id
+    current_user = postgres_db.query(Subscriber).where(Subscriber.id == user_id).first()
+
+    if current_user and current_user.active:
+        for field in fields:
+            value = current_user.__dict__[field]
+            if not value:
+                continue
+            context.user_data[field] = value
```

Comment:
так не будет проще? меньше вложенность
```suggestion
        context.user_data.update({
            field: val 
            for field in fields 
            if (val := current_user.__dict__[field])
        })
```